feat: Rework the backup and restore process (#478)
CluedIn-io/CluedIn.Documentation: docs/190-paas-operations/002-backup-and-restore.md

The process works by leveraging the automation account deployed to your managed resource group to run two main runbooks. One runbook will backup helm, and the other will backup the databases.

During the backup run, it will temporarily shut down the instance by scaling down all the pods running in Kubernetes to prevent any potential data loss. It will then take snapshots of each persistent disk and place these inside the managed resource group by default.

Once a backup has happened, it will reduce total snapshots to the supplied retention count (default is 7), starting with the oldest existing snapshot before scaling CluedIn up.

This process takes roughly 20 minutes to complete and is highly recommended to run out of hours.

## Setup the schedules
1. Navigate to the automation account located in the managed resource group of the environment you want to take backups. 
1. On the left hand side, click on **Schedules**
1. Click on **Add a schedule** and create it with the appropriate time, time zone, and days to run the backup and then click **Create**.

   ![backup-schedule](../../assets/images/ama/howtos/backup-schedule.png)

> 1. On the left hand side, click on **Runbooks**
> 1. Click on `backup-helm-values` and then schedules on the left side.
> 1. Click on **Add a schedule** and select the one just created and fill in the parameters.
> 1. Repeat the process for `backup-cluedin`

Reviewer: ```suggestion
1. On the left side, select **Runbooks**
1. Select `backup-helm-values`. Then, on the left side, select **Schedules**.
1. Select **Add a schedule**, and then select the schedule that you've just created. Fill in the parameters.
1. Repeat the process for `backup-cluedin`.
```

> Once this has been set, the automation account should proceed to backup as per your configuration.

Reviewer: ```suggestion
Once the schedules are set up, the automation account should proceed to run backup as per your configuration.
```

> For any further information on this, please reach out to CluedIn support.

Reviewer: ```suggestion
For any further information about backup, reach out to CluedIn support.
```

> # Restore
> If you're ever in a situation where you need to restore an environment, below will only cover an in-place restore, and not a disaster recovery to another region. It is highly recommended to reach out to CluedIn support in the first instance. 

Reviewer: ```suggestion
# Restore an environment
If you're ever in a situation where you need to restore an environment, this section will only cover an in-place restore, and not a disaster recovery to another region. It is highly recommended to reach out to CluedIn support in the first instance. 
```

> ## Restore flow
> 1. Scale down the pods on your AKS so no CluedIn pods are running.
> 1. Restore the disks located in the snapshot location over top of the existing PVCs.

Reviewer: ```suggestion
**To restore an environment**
1. Scale down the pods on your AKS so no CluedIn pods are running.
1. Restore the disks in the snapshot location over top of the existing PVCs.
```

   {:.important}
   The disk name and tags must match during a restore, otherwise you may run into problems during operation.
   All restored disks must be from the same time. There may be a few minutes between each disk as it runs sequentially.

> 1. Once all disks have been restored from the same time , it is then safe to scale the cluster back up.
> 1. After 5 minutes, the instance should then be rolled back.

Reviewer: ```suggestion
1. Once all disks have been restored from the same time, proceed to scale the cluster back up.
 
     After 5 minutes, the instance should then be rolled back.
```

# Further information

> If you require further information at any point regarding the above, please reach out to CluedIn support who will assist.

Reviewer: I think this should be deleted. CluedIn support is mentioned in several places in the article, there is no need to repeat it.

Author: Removed
